conf/test_all.config

```diff
@@ -33,6 +33,7 @@ params {
     phase        = true
     normalize    = true
     compute_freq = false
+    chunk_model  = "recursive"
```

Comment:
What is the rationale behind this? Why adding this now and in this PR? We shouldn't be making a lot of new changes or additions before the first release PR. We are supposed to correct the issues/changes required by reviewer. The problem with adding new features now is that:
- Increases the scope of this PR, making it difficult for me to review all new changes all the time. 
- We would have to test all of these new features, add nf-tests, etc. 

If the full chromosome test passed without these changes, then it is fine, we shouldn't add any new features (at least in this PR or stage).

Reply:
This is due to the fulltest not passing due to an overlapp in the chunks.
I've got an error at the ligation step that said:

`Three files overlapping at position: 10804701`

I've looked into the issue of the Glimpse2 repository and the advice was maybe to change the chunk model (we first went with `--recursive` as `--sequential` was not working on small region) and to increase the minimal window size.

Reply:
I've launched back the test on my HPC but it's really slow as my HPC is full.

Reply:
I would say to stick to glimpse1 then, which we know was working for the full chromosome test. Then in a **separate** PR we can review if changes to glimpse2 are necessary.

Reply:
I can create a separate PR for this if you prefer.